src/features/Courses/CoursesPage/index.jsx

```diff
@@ -52,9 +52,7 @@ const CoursesPage = () => {
   };
 
   useEffect(() => {
-    if (id > 0) {
-      fetchData(filters);
-    }
+    fetchData(filters);
```

Comment:
Did you check that in the first render there are not unnecessary calls to this service?  Also in the case that some of the dependencies changed?

Reply:
@AuraAlba Yes, the same calls are made as before.